### PR TITLE
Create eoseco.yml

### PR DESCRIPTION
**Pull request by qiushaoxi**

### CODE REVIEW

File: block-producers/scholar-testnet/eoseco.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - public_key: EOS85fDTxvb2P6cXuz5nQyaDwTPbemwcw6KyVDmHHZaGFNog5fBCK
+      weight: 1
+  active:
+    threshold: 1
+    keys:
+    - public_key: EOS85fDTxvb2P6cXuz5nQyaDwTPbemwcw6KyVDmHHZaGFNog5fBCK
+      weight: 1
+
+# Server Config (Optional)
+eosio_http_host: scholar.eoseco.com
```
Comment (DenisCarriere) [inline block-producers/scholar-testnet/eoseco.yml]: 👎 Domain & port doesn't work:

http://scholar.eoseco.com:8888/v1/chain/get_info

File: block-producers/scholar-testnet/eoseco.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# Encryption (Optional)
+pgp_public_key: 
+
+# Social (Optional)
+social_twitter: https://twitter.com/Eoseco2018 
+social_telegram: t.me/eoseco
+social_facebook: https://www.facebook.com/eoseco2018 
+social_github: https://github.com/Eoseco
+social_youtube: 
+social_keybase: eoseco
```
Comment (DenisCarriere) [inline block-producers/scholar-testnet/eoseco.yml]: ❌ Social account must use a fully qualified domain URL

https://keybase.io/eoseco

File: block-producers/scholar-testnet/eoseco.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+eosio_https_port:
+eosio_p2p_host: scholar.eoseco.com
+eosio_p2p_port: 9876
+
+# Encryption (Optional)
+pgp_public_key: 
+
+# Social (Optional)
+social_twitter: https://twitter.com/Eoseco2018 
+social_telegram: t.me/eoseco
```
Comment (DenisCarriere) [inline block-producers/scholar-testnet/eoseco.yml]: ❌ Social account must use a fully qualified domain URL

https://t.me/eoseco

File: block-producers/scholar-testnet/eoseco.yml (hunk 1 above)
Comment (qiushaoxi) [reply block-producers/scholar-testnet/eoseco.yml]: sorry, i will request after setting dns

Comment (DenisCarriere) [reply block-producers/scholar-testnet/eoseco.yml]: 👍 Ok! Sounds good, just let me know when it's up and running as no-producing node, once the node is connected with a producer name, i can update the Scholar Testnet to include you

Comment (qiushaoxi) [reply block-producers/scholar-testnet/eoseco.yml]: please check agian

**Pull request merged**